### DISCUSSION TITLE
Code block: I have white empty row adding both file name and line numbers

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#4216] [Q&A]
Hi,
if I add this in my md file

````
``` txt linenums="1" title="input_00.csv"
text,number,date,,type
Lorem,37.256,2022-01-13,012543,A
,,,,
,Ciao,2023-01-13,78956,C
```
````

I have a white empty row

![image](https://user-images.githubusercontent.com/30607/183244352-44bc349e-161b-4938-955c-ad3737ff8132.png)

Is it normal? Am I using some wrong syntax?

Thank you

**Comment by squidfunk** [comment]
The syntax looks correct. Make sure that this is not related to your customizations. If you can provide a minimal reproducible example on a clean Material for MkDocs installation, you an create an issue and I'll look into it.

**Reply by aborruso** [reply]
> The syntax looks correct. Make sure that this is not related to your customizations. If you can provide a minimal reproducible example on a clean Material for MkDocs installation, you an create an issue and I'll look into it.

it works in a clean installation, than it's an error of mine. Thank you

**Reply by aborruso** [reply]
> The syntax looks correct. Make sure that this is not related to your customizations. If you can provide a minimal reproducible example on a clean Material for MkDocs installation, you an create an issue and I'll look into it.

@squidfunk  I discovered the problem: if I add [sortable tables](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables), I have that white row (and inside it I have the sort button).

I think that it shouldn't break the code formatting, but it does.

**Reply by squidfunk** [reply]
Please create an issue with a minimum reproducible example if it is related to the way we recommend setting up sortable tables.
